app/views/auth/ImportAccountView/ImportAccountForm.tsx

```diff
@@ -96,12 +98,26 @@ const ImportAccountForm: FC<ImportAccountFormProps> = ({
     submit: null,
   };
 
+  const validatePassPhrase = (pass: string | undefined): boolean => {
```

Comment:
I find the naming `passphrase` to be a bit confusing here. It's more of an mnemonic device to remember the entropy than an actual passphrase created by the user. I think we should call this `mnemonicEntropy`